### PR TITLE
Polish translations for task A

### PR DESCRIPTION
**Pull request by MilyMilo**
Everything should be fine by now.

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@MilyMilo, it looks like you've already signed the Dropbox CLA. Thanks!

File: translations/Polish/research/notes.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+General notes:
+
+* The main difference between Polish and English writing are the special letters, in English, there is no ą, ć, ę, ł, ń, ś, ó, ż, ź.
```
Comment (trueskawka) [inline translations/Polish/research/notes.md]: In general we try to keep line lengths at <= 76 characters, could you shorten them?

File: translations/Polish/research/notes.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+* Polish grammar differs much from the English. What matters in the context of websites, texts are usually longer in Polish.
+* Polish and English are both written from left to right.
+
+Thoughts after analysing the screenshots:
+ 
+* Polish words and sentences are quite often longer what forces sites to scale. For instance: Google_preferences, what fitted on one site displayed in English, I had to scroll in Polish.
```
Comment (trueskawka) [inline translations/Polish/research/notes.md]: I'd say `which forces sites`.

Not sure what you mean by `scale` here, did you mean `adjust`?

`For instance in Enligsh Google_preferences fit above the fold, but I had to scroll the site in Polish`

File: translations/Polish/research/notes.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+* Sometimes fonts don't support polish special letters and therefore break site's look (e.g. Instagram)
```
Comment (trueskawka) [inline translations/Polish/research/notes.md]: `Polish`, `special characters`, missing full stop.

File: translations/Polish/research/notes.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+* Some sites are inconsistent. Content in different languages differs. Same applies to design, on Wikipedia for example home page, has a different header.
```
Comment (trueskawka) [inline translations/Polish/research/notes.md]: `for example Wikipedia home page has a different header`

File: translations/Polish/research/notes.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+General notes:
+
+* The main difference between Polish and English writing are the special letters, in English, there is no ą, ć, ę, ł, ń, ś, ó, ż, ź.
+* Polish grammar differs much from the English. What matters in the context of websites, texts are usually longer in Polish.
+* Polish and English are both written from left to right.
```
Comment (trueskawka) [inline translations/Polish/research/notes.md]: I don't think you've addressed these questions here:

> Think about the language used - is it more formal or informal? Are there any specific forms that should or shouldn't be used? What words or phrases are most commonly used? What could sound awkward?

**Comment by trueskawka** [comment]
Hi @MilyMilo!

Thanks for working on this! 

You've done some pretty thorough research and I think after the next round of changes it'll be ready for the final review. Could you address my comments?

Also, learning more about the git flow will be beneficial for you :) Some helpful docs on git and editing your commit history:

- [Git guide](http://zulip.readthedocs.io/en/latest/git-guide.html)
- [working copies](https://github.com/zulip/zulip-gci/blob/master/docs/working-copies.md)
- [fixing commits](https://github.com/zulip/zulip-gci/blob/master/docs/fixing-commits.md)

Thanks, Miłosz!

@TomaszKolek Up to you :)

**Comment by MilyMilo** [comment]
Right, done. Hope it's fine now :) I'm sorry that you had to correct it so many times. These were my first translations and although I think it wasn't that bad I'll have to learn much more and for sure gain experience. I've also overdone few times, tried to use more fancy words and you know what happened. Hope you understand :)

File: translations/Polish/research/notes.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+Thoughts after analysing the screenshots:
+ 
+* Polish words and sentences are quite often longer which forces sites to 
+adjust. For instance in English Google_preferences fit above the fold, but
+I had to scroll the site in Polish. 
+* Sometimes fonts don't support Polish special characters and therefore 
+break site's look (e.g. Instagram).
+* Some sites are inconsistent. Content in different languages differs. Same
+ applies to design, for example Wikipedia home page has a different header
+* Comparing formality sites did not really differ, content kept it's style.
```
Comment (trueskawka) [inline translations/Polish/research/notes.md]: `its style`

File: translations/Polish/research/notes.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+
+The style used, was rather informal than formal, although it wasn't
+completely informal. Some sites like Wikipedia had more formal style than
+for example twitter's home page. Also more serious, I'd say sections, were
```
Comment (trueskawka) [inline translations/Polish/research/notes.md]: `Twitter`

Comment (trueskawka) [inline translations/Polish/research/notes.md]: `Also more formal sections` - I don't think `serious` fits here.

File: translations/Polish/research/notes.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+written more formally (e.g. Settings on facebook).
```
Comment (trueskawka) [inline translations/Polish/research/notes.md]: What made them more formal?

`Facebook`

File: translations/Polish/research/notes.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+
+Formal forms such like Mr., Mrs., Sir or Madam weren't used on any of 
+the analysed sites and in my opinion they shouldn't be used because then 
+text would sound too formal (of course there will be some exceptions).
+Quite balanced language fits best almost in any context so that's why
```
Comment (trueskawka) [inline translations/Polish/research/notes.md]: `in almost any context`

**Comment by trueskawka** [comment]
Hi @MilyMilo!

Thanks for the improvements.

Could you address my new comments? 

Also, could you squash your commits and make sure the commit message is the same as in the task description - `translations: Research translations for <language>.`?

Thanks!

**Comment by trueskawka** [comment]
Thanks for improving @MilyMilo! Merging :)

**Pull request merged**